src/groups/mqb/mqbnet/mqbnet_tcpsessionfactory.cpp

```diff
@@ -262,25 +262,6 @@ struct PortMatcher {
         return listener.port() == d_port;
     }
 };
-
-template <typename T>
-T* channelCast(bmqio::Channel* base)
```

Comment:
This was just dead code we never removed? Happy to see it removed :)

Reply:
This might be related to TLS change, where we were needed to find a concrete `bmqio::Channel` implementation within the chain of nested `bmqio::Channel`s